code/game/objects/effects/mainttraps.dm

```diff
@@ -126,6 +126,35 @@
 	inuse = FALSE
 	return TRUE
 
+/obj/effect/trap/nexus/doorbolt/wikipedia
```

Comment:
if this is a subtype, why are you copying all the code? just make a separate trap that plays sounds

Reply:
i am stupid is the only answer